### PR TITLE
Add no_alloc compatibility

### LINKED ISSUE #2: no_alloc not supported
I've been trying to use this library in a `no_std` environment, but I'm running into a compile time `no global memory allocator error` with `test` disabled. This appears to be due to the use of `alloc::borrow::ToOwned`. Do I need to configure something else in my environment? 

I'm opening a PR with a small patch that I've been using for the past month -- feel free to ignore if it is indeed a configuration issue.

### PR DESCRIPTION
**Pull request by ayushagrawal2**
Should fix #2 -- I haven't run into any issues yet in my usage over the past month.

### CODE REVIEW

File: src/expander.rs [hunk 1]
```diff
@@ -335,14 +334,12 @@ where
...
-                self.address,
-                &[COMMAND_CONF_0, self.configuration_0.as_value().to_owned()],
-            ),
-            Bank::Bank1 => self.bus.write(
-                self.address,
-                &[COMMAND_CONF_1, self.configuration_1.as_value().to_owned()],
-            ),
+            Bank::Bank0 => self
+                .bus
+                .write(self.address, &[COMMAND_CONF_0, self.configuration_0.as_value().clone()]),
```
Comment (marius-meissner) [inline src/expander.rs]: As u8 implements the Copy trait, no cloning is required, as also suggested by Clippy.
Please try to dereference/copy instead:
```rust
*self.configuration_0.as_value()
```

This also applies to lines 342, 351, 354, 363 and 366.

**Review by marius-meissner — CHANGES_REQUESTED** [review]
Thank you! Looks good, just one minor suggestion.
Seems QA action pipeline is currently not triggered by PR.
Could you please adjust
https://github.com/atlas-aero/rt-PCA9539/blob/366bc6e345391a06eb4e5ae27709879cfa976bd2/.github/workflows/qa.yaml#L3 to 
```yaml
on:
  push:
  pull_request:
```

**Comment by ayushagrawal2** [comment]
Thanks for the feedback! Let me know if any further changes are needed

**Comment by marius-meissner** [comment]
LGTM

**Pull request merged**